README.md

```diff
@@ -69,6 +71,14 @@ bandstructure_flow = RelaxBandStructureMaker().make(mgo_structure)
 run_locally(bandstructure_flow, create_folders=True)
 ```
 
+Before the above code can run successfully, you'll need to
+
+- set `PMG_VASP_PSP_DIR` in `~/.pmgrc.yaml` to a directory with VASP POTCAR files,
+- tell atomate2 through `ATOMATE2_VASP_CMD=/path/to/vasp_std` where to find your VASP binary,
+- have a MongoDB instance ready to accept job output.
+
+See the [installation] steps for details how to set all of this up.
```

Comment:
How's this? The flow from this line to the next paragraph is a bit awkward maybe? But wanted to keep it short.

Reply:
Could you remove the `ATOMATE2_VASP_CMD=/path/to/vasp_std` bit or change it to `export ATOMATE2_VASP_CMD=/path/to/vasp_std`.

Thanks!